packages/react-vapor/src/components/textInput/useForm.ts

```diff
@@ -0,0 +1,15 @@
+import {useContext, useMemo, Dispatch} from 'react';
+import {TextInputAction, TextInputState} from './TextInputReducer';
+import {FormComponent, FormContext} from '../form/FormProvider';
+
+export const useFormComponent = (
+    component: FormComponent
+): {state: Record<string, TextInputState>; dispatch: (id: string, action: TextInputAction) => void} => {
```

Comment:
I think this dispatch should provide thinks to make it easier to dispatch to multiple inputs at the time.

```ts
dispatch: (ids: string[] | 'All', action) => void
```

Reply:
Here the dispatch should be able to dispatch other actions than TextInputActions

Reply:
FYI: since we cannot pinpoint a use case for a dispatch function, we will leave it as it is right now, so with only the state returned.